go.mod

```diff
@@ -9,14 +9,13 @@ require (
 	github.com/docker/docker v1.13.1
 	github.com/docker/go-connections v0.4.0 // indirect
 	github.com/docker/go-units v0.4.0 // indirect
-	github.com/fsnotify/fsnotify v1.4.7 // indirect
 	github.com/hpcloud/tail v1.0.0
```

Comment:
Is this still required? If it's removed, does `go mod` add it back as an indirect?

Reply:
removed